docs/components/web_wallet.md

```diff
@@ -34,3 +34,363 @@ User agent web wallet supports various wallet operations and also [DIDComm](http
 - **Presentation Exchange**: This operation can be used by a website (a Relying Party or verifier) to request submission of proof from wallet in align with requester's [proof requirements](https://identity.foundation/presentation-exchange/). This request can also be made in conjuction with DID Connect operation, so that wallet can request further information from requester (a Relying party or verifier) over DID comm channel.
 
+## Login to wallet
+After successful login, web wallet will load [polyfill library](https://github.com/digitalbazaar/credential-handler-polyfill) and a web credential handler will be installed which will serve all your credential handler request as per [Credential Handler API](https://w3c-ccg.github.io/credential-handler-api/) (aka CHAPI) standards.
+
+In addition to that, user agent web wallet will perform some onboarding operations during first login of an user to support various DID comm operations which includes,
+
+ - Registering with router(also known as mediator). Since user agent web wallet is technically also an agent which runs in your browser. Since it has no inbound transport and cannot be online always, it has to register 
+ with [router](https://github.com/hyperledger/aries-framework-go/blob/master/docs/didcomm_mediator.md) to route the messages to it by asking for permission. On successful grant, agent receives the endpoint and routing key details.
+ Sample configuration of how to pass a mediator URL in user agent setup can be found [here](https://github.com/trustbloc/edge-agent/blob/master/test/bdd/fixtures/agent-wasm/docker-compose.yml#L71)
+ - Creating TrustBloc DID. By default user agent web wallet uses trustbloc DIDs for generating verifiable presentation response over CHAPI.
+ - Saving created DIDs and other metadata in storage for future operations.
+
+## Storing credential(s) in wallet
+A single or multiple credentials can be stored into wallet through `store()` call using a CHAPI request as given in sample below. 
+
+In below sample, a presentation with single or multiple credentials is stored into wallet,
+
+ ```
+ const webCredential = new WebCredential('VerifiablePresentation', {
+   '@context': 'https://www.w3.org/2018/credentials/v1',
+   ...presentation
+ });
+
+const result = await navigator.credentials.store(webCredential);
+if(!result) {
+  console.log('store credential operation did not succeed');
+}
+
+```
+
+Note: while storing, credential(s) always has to be in presentation format 
+
+## Getting credential from wallet
+Credentials can be requested from web wallet through `get()` call using a CHAPI request as given in sample below,
+
+```javascript
+let credentialQuery = {
+  "web": {
+    "VerifiablePresentation": {
+      "query": [
+        {
+          "type": "QueryByExample",
```

Comment:
This type is not supported by the wallet as far as I can see.

We should put more representative examples and point to their documentation.